packages/custom-functions-metadata/src/cli.ts

```diff
@@ -4,7 +4,7 @@
 // Licensed under the MIT license.
 
 import * as commander from "commander";
-import { logErrorMessage } from "office-addin-cli";
+import { logErrorMessage } from "office-addin-usage-data";
```

Comment:
Why are you doing this? It shouldn't be part of office-addin-usage-data, as it has nothing to do with telemetry. It's also a breaking change for all of the package.

Reply:
Agree that `usage-data` is probably not the best name for the package, but I think it makes more sense to have the `logError` command than `office-addin-cli`.
We were thinking of changing its name to `logging` or something like that later on.

Yeah, it would be a breaking change from `office-addin-cli`, but why would it break other packages?

Reply:
Dependencies . . . packages that depend on cli to get logError will need to depend on usage-data instead (which our probably do anyway . . . but not necessarily customers).  That is what this will be a major version update for the packages.

Reply:
But this package depends on `cli`, we can just change the dependencies of it internally. There would be no external consequences that clients would see, we are just changing some dependencies of the package.

It would need a version bump, but I don't get why it would be major if there wouldn't be any breaking changes to external clients.

Reply:
I'm not sure what you mean by "this package depends on cli".

In general any current project can have a dependency on cli or usage-data or both and we don't have control over that.  I'm not sure how introducing a dependency in cli on usage-data will prevent breaks.  Even if the package is installed because we depend on it, it doesn't mean the reference in code can get logError from usage-data by way of cli.

Reply:
Yeah, I agree that `office-addin-cli` would have breaking changes as `logError` is not there anymore.

What I don't get is how another package, such as `office-addin-debugging` which used to depend on `office-addin-cli` would have breaking changes. We pretty much only changed the dependencies, with no external signatures changing.

Reply:
Just like in this PR . . . you have to change package.json to get the usage-data package if it doesn't already get it and you have to change the import statement in commands.ts (and other similar files) which currently says 'import { logErrorMessage, parseNumber, getPackageJsonScript } from "office-addin-cli";' but would need to use the updated package structure.  If you simply reved the cli version the build would break.  

This PR does that for our packages in this repo, but that doesn't automatically do if for other repos and or project.  If you didn't update the major version of the debugging package and npm installed it then it would ask for the new major version of the cli package and the project may have references to the cli separately that wouldn't be ready for this update.

There may be something I don't understand about npm package management that might take care of this automatically (having both version of the cli package?), but I'm currently under the impression that both should be major version bumps.

Reply:
Makes sense, thanks for the explanation